feat(Tag): Show selected tags list

geoffreykwan opened the pull request:
## Changes
- We now show the list of selected tags to filter by
- Added clear filters button

## Test
- Make sure tags that are selected using the tags filter drop down show up in the list
- Make sure removing a tag using the tag x button removes the tag from the list and also unselects it in the tags filter drop down
- Make sure the clear filters button unselects all the tags to filter by

geoffreykwan committed:
feat(Tag): Show selected tags list

breity committed:
Add keyboard accessibility for remove filter buttons.
Update mat-chip label font-size to match buttons.

breity reviewed (changes requested):
Functionality works well. I added keyboard accessibility to the remove filter 'x' buttons and also updated the mat-chip label font-size to match mat-button font-size.

As discussed, I think we can take out the "Clear filters" option added in this set of changes and use the existing "Clear filters" option that is visible when a search term has been entered. So that option would appear when any tag or search filters are present and would clear out both.

geoffreykwan committed:
feat(Tag): Clear tags using the existing clear filters link

Also fix message when there are no active runs or no archived runs.

geoffreykwan commented:
Ok, I've changed it so that we use the "Clear filters" link to clear out the tags. I also fixed the messages when there are no active runs or no archived runs.

geoffreykwan committed:
Merge branch 'issue-1686-implement-unit-tagging' into show-selected-tags-list

breity reviewed (approved):
Looks good. 👍
